modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/ScaledFloatFieldMapper.java

```diff
@@ -713,32 +712,19 @@ public int docValueCount() {
     }
 
     @Override
-    protected SyntheticSourceMode syntheticSourceMode() {
-        return SyntheticSourceMode.NATIVE;
-    }
+    protected SyntheticSourceSupport syntheticSourceSupport() {
+        if (hasDocValues) {
```

Comment:
Doesn't this override storing the source as-is when `copy_to` is set?

Reply:
When `copy_to` is set this won't even be called. That's the idea of `syntheticSourceSupport` introduction.

Reply:
I see, this is from the `final` in `FieldMapper` right? Nice.

Reply:
yep